siege_utilities/geo/spatial_data.py

```
@@ -626,26 +667,32 @@ def _construct_filename_with_fips_validation(self, year: int, boundary_type: str
-                return patterns['filename_patterns']['congress'].format(
-                    year=year, congress_num=congress_number
-                )
+                # Congress number provided separately — zero-pad to 3 digits
+                congress_num = f"{congress_number:03d}"
             else:
                 raise BoundaryConfigurationError(
                     f"Congressional district boundary type '{boundary_type}' requires a "
                     f"congress number. Provide it via the boundary_type name (e.g., 'cd118') "
                     f"or the congress_number parameter.",
                     context={"boundary_type": boundary_type, "year": year},
                 )
```

Ruff (0.15.11)

[warning] Avoid specifying long messages outside the exception class

(TRY003)